package/README.md

```diff
@@ -71,6 +71,17 @@ You can then use that `getClient` function in your server components:
 const { data } = await getClient().query({ query: userQuery });
 ```
 
+#### Multiple clients
+
+If you need to manage multiple clients, for example if you have different endpoints for content previews,
+the function you pass to `registerApolloClient` can accept arbitrary arguments.
+If you're using Typescript, you can hint their types as a tuple:
```

Comment:
```suggestion
you can pass a function to `registerApolloClient` that accepts arguments.
```

I'd suggest removing that last sentence. I think the code sample does a good job showing off how to use TypeScript with this function, so this point is inferred. I also had a suggestion for a slight tweak in that other sentence that gives it more active voice.